package.json

```diff
@@ -18,7 +18,7 @@
     "test": "npx jest --silent"
   },
   "peerDependencies": {
-    "@sap/cds": ">=7"
+    "@sap/cds": "*"
```

Comment:
```suggestion
    "@sap/cds": ">=7"
```

Reply:
I don't like having that. In the worst case, @sap/cds is installed twice. I'd rather have a version check in the plugin itself.

Reply:
reverted back